java-client/src/generated/java/org/opensearch/client/opensearch/indices/AddBlockRequest.java

```diff
@@ -46,134 +53,145 @@
 import org.opensearch.client.transport.Endpoint;
 import org.opensearch.client.transport.endpoints.SimpleEndpoint;
 import org.opensearch.client.util.ApiTypeHelper;
+import org.opensearch.client.util.CopyableBuilder;
 import org.opensearch.client.util.ObjectBuilder;
 import org.opensearch.client.util.ObjectBuilderBase;
+import org.opensearch.client.util.ToCopyableBuilder;
 
 // typedef: indices.add_block.Request
 
 /**
  * Adds a block to an index.
- *
  */
+@Generated("org.opensearch.client.codegen.CodeGenerator")
+public class AddBlockRequest extends RequestBase implements ToCopyableBuilder<AddBlockRequest.Builder, AddBlockRequest> {
 
-public class AddBlockRequest extends RequestBase {
     @Nullable
     private final Boolean allowNoIndices;
 
+    @Nonnull
     private final IndicesBlockOptions block;
 
+    @Nullable
+    private final Time clusterManagerTimeout;
+
+    @Nonnull
     private final List<ExpandWildcard> expandWildcards;
 
     @Nullable
     private final Boolean ignoreUnavailable;
 
-    private final String index;
+    @Nonnull
+    private final List<String> index;
```

Comment:
Hm ... this seems to be out of sync, the name should be `indices` as per https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/action/admin/indices/readonly/AddIndexBlockRequest.java#L59 ?

Reply:
The spec copies the naming from the REST handler: https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/rest/action/admin/indices/RestAddIndexBlockAction.java

Other operations which already accepted a list like [GetIndexRequest](https://github.com/opensearch-project/opensearch-java/blob/dc41a99f1fd76fa59db8aa28c5466602cbcc6eb1/java-client/src/main/java/org/opensearch/client/opensearch/indices/GetIndexRequest.java) still refer to it as `index`